cmake/Modules/MacroQbtCommonConfig.cmake

```diff
@@ -33,14 +33,18 @@ macro(qbt_common_config)
 
     if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
         target_compile_definitions(qbt_common_cfg INTERFACE
-            NTDDI_VERSION=0x06010000
-            _WIN32_WINNT=0x0601
-            _WIN32_IE=0x0601
             WIN32_LEAN_AND_MEAN
             NOMINMAX
             UNICODE
             _UNICODE
         )
+        if (NOT QT6)
```

Comment:
I still think this doesn't apply to all cases, I would suggest using something like `if (NOT msys2)`.

Reply:
Alright, understood. msys2 should not be a special case. If this is msys2/mingw specific case I should add a patch there. Thank you for the review ❤️

Reply:
Just letting you know, we (or at least me) would accept platform specific patches.